unittests/savanna_disaster_recovery.cpp

```diff
@@ -0,0 +1,270 @@
+#include "savanna_cluster.hpp"
+
+using namespace eosio::chain;
+using namespace eosio::testing;
+
```

Comment:
The PR description is very informative. Copy some high level info to the beginning of each test to describe test purpose.

Reply:
I tried to have the test be almost as readable as the list of steps in the description above, and I feel it is better to have one source of truth.

When duplicating the code logic into comments, they often become inconsistent as the code is changed while the comments are not. This is why I did not want to add the list of steps as a block comment on top. However, let me see if I can add a more general comment describing the intent of the test.

Reply:
Thanks!